README.md

```diff
@@ -65,21 +65,19 @@ to DOM-world and is used on top level of application. It is peer dependency of
 * `react-test-renderer` - render React components to pure JavaScript objects.
 * `jest` - testing framework to write unit specs (including snapshots).
 * `jest-image-snapshot` - library provides ability to check difference between two image
-
+* `jest-styled-components` - a set of utilities for testing Styled Components with Jest.
+* `styled-components` - requirement of `jest-styled-components`.
+* `@eigenspace/jest-testing-utils` - contains jest helpers and patchers.
 
 # Why do we have that dev dependencies?
 
 * `@eigenspace/codestyle` - includes tslint rules, config for typescript.
 * `@types/*` - contains type definitions for specific library.
+* `@eigenspace/common-types` - contains common type definitions for our libraries.
 * `ts-jest` - it lets you use Jest to test projects written in TypeScript.
-* `ts-loader` - it is used to load typescript code with webpack. 
 * `tslint` - it checks TypeScript code for readability, maintainability, and functionality errors.
 * `typescript` - is a superset of JavaScript that have static type-checking and ECMAScript features.
-See `webpack.config.js`.
-* `webpack` - it create app bundle for dev mode and production. 
-* `copy-webpack-plugin` - used for copy package.json in package bundle.
-* `clean-webpack-plugin` - used for clean bundle before run building.
 * `husky` - used for configure git hooks.
 * `lint-staged` - used for configure linters against staged git files.
 * `whatwg-fetch` - this project is a polyfill for `window.fetch`.
-* `webpack-cli` - command line interface dor webpack.
\ No newline at end of file
+* `@eigenspace/helper-scripts` - common scripts for dev. environment.
```

Comment:
I suggest to add description of deps in the same order we have them in `package.json`.

Reply:
We have a task for working with README (https://github.com/eigen-space/codestyle/issues/6) We can add it to sort the dependencies in this list